### PR TITLE
Updating promotion space with the mailing list

### PR DESCRIPTION
**Pull request by Tomcowan1**
### Trello card

https://trello.com/c/00caDBym/2450-replace-events-promo-on-home-page-to-highlight-mailing-list-tie-in-with-new-tv-campaign

### CODE REVIEW

**Comment by gemmadallmandfe** [comment]
Checks on this PR have failed @peteryates - is this due to the wider github issues?

@Tomcowan1 I will review properly when available in the Review app but think the links still need to be changed too?

**Comment by Tomcowan1** [comment]
Ah yes @gemmadallmandfe forgot to do that. Have edited

**Comment by peteryates** [comment]
Looking at the earlier failures it does look like it was just a issue elsewhere and not with the build itself @gemmadallmandfe 

```
buildx failed with: error: failed to solve: failed to fetch oauth token: unexpected status: 503 Service Unavailable
```

File: app/views/content/ways-to-train.md [hunk 1]
```diff
@@ -3,22 +3,13 @@ title: "Ways to train"
...
-subtitle_button: "Book your place"
-subtitle_link: "event-categories/train-to-teach-events"
+subtitle: Get information and support to help you become a teacher. 
+subtitle_button: "Register for updates"
+subtitle_link: "/mailinglist/signup/name"
 date: "2021-06-07"
 backlink: "../"
 navigation: 20
 article_classes: ['ways-to-train']
-right_column:
```
Comment (gemmadallmandfe) [inline app/views/content/ways-to-train.md:13]: Do we need to remove the sidebar CTA for events as well?

File: app/views/content/funding-your-training.md [hunk 2]
```diff
@@ -5,20 +5,12 @@
...
-  subtitle: New events added - join an event near you and get all the info you need to become a teacher. 
-  subtitle_button: "Book your place"
-  subtitle_link: "/event-categories/train-to-teach-events"
+  subtitle: Get information and support to help you become a teacher. 
+  subtitle_button: "Register for updates"
+  subtitle_link: "/mailinglist/signup/name"
   backlink: "../"
   navigation: 25
   right_column:
-  ctas:
```
Comment (gemmadallmandfe) [inline app/views/content/funding-your-training.md:14]: Weirdly, I can't actually see this sidebar CTA in production (why is that @peteryates ?)

(If it was working) Do we need to remove this events CTA as well as changing the top promo?

Comment (peteryates) [reply app/views/content/funding-your-training.md:14]: Ah the indentation isn't quite right there, `cts` should be inside `right_column`. Fixed it separately in #1779

Comment (Tomcowan1) [reply app/views/content/funding-your-training.md:14]: @gemmadallmandfe I would remove the events right hand CTA. Originally it was only there for the promotional space and (I _think_ )the stats show that it isn't used V much.

Comment (gemmadallmandfe) [reply app/views/content/funding-your-training.md:14]: I've looked in GA and it doesn't look well used so happy to remove

File: app/views/content/ways-to-train.md (hunk 1 above)
Comment (gemmadallmandfe) [reply app/views/content/ways-to-train.md:13]: Happy for this to be removed having looked in GA

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me - happy for this to be merged.

@peteryates does this have any dependency on 1779 or can it be merged now?

**Comment by peteryates** [comment]
No dependency - they can both be merged indepndently

**Comment by peteryates** [comment]
Rebased (after #1779) and repushed

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me - happy for this to be merged

FYI @Tomcowan1 - the sidebar CTA for events is still on Funding due to the fix @peteryates did in 1779. Happy to it to be removed in a separate PR if necessary but didn't want to hold this one up any longer!

**Pull request merged**